scripts/format-code.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
+
+VERSION=1.13.0
+JARFILE=google-java-format-$VERSION-all-deps.jar
+
+mkdir -p .cache
```

Comment:
Use variable for cache directory?

So the last command becomes:
```
java ...  -jar $CACHE_DIR/$JARFILE --replace $changed_java_files
```

Reply:
I don't think we will ever change this variable.